model-optimizer/extensions/front/onnx/elementwise_ext.py

```diff
@@ -188,3 +188,13 @@ class XorExtractor(FrontExtractorOp):
     def extract(cls, node):
         LogicalXor.update_node_stat(node)
         return cls.enabled
+
+
+class RoundFrontExtractor(FrontExtractorOp):
```

Comment:
Add extractor for mxnet Round: https://mxnet.apache.org/versions/1.7.0/api/python/docs/api/symbol/symbol.html#mxnet.symbol.round

Reply:
Done, extractor added.